update.py

```diff
@@ -63,7 +64,9 @@ def check_notices(session, sessionData):
             notice['attachment_url'] = ERP_ATTACHMENT_URL + m.group(1)
             r = session.get(notice['attachment_url'], stream=True)
             r.raw.decode_content = True
+            notice['attachment_raw'] = r
             hash_ = hashlib.md5()
+
             for chunk in iter(lambda: r.raw.read(4096), b""):
```

Comment:
If `r.raw` is a stream, are we allowed to read it twice (once here and once in the mailer), or does it get consumed the first time you read it?

Reply:
It gets consumed only after calling `r.raw.read()` - `r.raw` can be used twice.